sdk/storage/inc/common/common_request_policy.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "http/policy.hpp"
+
+namespace Azure { namespace Storage {
+
+  class CommonRequestPolicy : public Core::Http::HttpPolicy {
```

Comment:
Nit, CommonRequest isn't a very descriptive name for a policy

Reply:
okay, but i cannot think of any better name for now.

Reply:
@katmsft any insights on this?

Reply:
CommanHeadersRequestPolicy/SharedHeadersRequestPolicy are maybe better? I'm not sure.

---
In reply to: [436615210](https://github.com/Azure/azure-sdk-for-cpp/pull/160#discussion_r436615210) [](ancestors = 436615210)

Reply:
@gilbertw i'm going to change them to `CommonHeadersRequestPolicy`. If you still have concern. please create an issue so that we can further discuss.

Reply:
@gilbertw I created an issue https://github.com/Azure/azure-sdk-for-cpp/issues/168 to track this.